tests/framework/CodeInterfaceTests/RELAP5/gold/RELAP5interfaceTestOperator/testDummyStep/2/typpwr2.i

```diff
@@ -0,0 +1,2034 @@
+*RAVEN INPUT VALUES
```

Comment:
Are we allowed to share this file?

Reply:
yes it is a test from RELAP...very basic and without "proprietary" info